### ISSUE TITLE
Tag "v.5.9.1" should be "v5.9.1"

### ISSUE THREAD

**Issue by terorie** [johnkerl/miller#366]
The latest alpine build is failing because it expects a release at tag `v5.9.1` for version 5.9.1.
Perhaps the existing tag `v.5.9.1` was meant to be called `v5.9.1`?

```
~/miller $ abuild checksum
>>> miller: Fetching miller-5.9.1.tar.gz::https://github.com/johnkerl/miller/archive/v5.9.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   123  100   123    0     0    467      0 --:--:-- --:--:-- --:--:--   467
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
>>> ERROR: miller: checksum failed
```

**Comment by johnkerl** [comment]
Yikes. Thanks. I thought I was fixing something but I had it backwards. :(

**Comment by johnkerl** [comment]
P.S. Found the issue -- originally I fumble-fingered `v.5.9.1` (note the first `.`), then "fixed" that to `5.9.1`, then (thanks to you) fixed that to `v5.9.1`.

PP.S. that was FAST!! ;)

**Comment by terorie** [comment]
@johnkerl Thanks for re-tagging, it's working now. 
> PP.S. that was FAST!! ;)

Could have been faster, I was 30 mins late. I'll look into automating releases to Alpine :smile:
